Vlad us 272

vladimircasap opened the pull request:
Positive TC Done

stasiukk reviewed (approved):
I ran your code,  and it looks great! Good job!

Haden-Z reviewed (changes requested):
Hi Vladimir,
Your codes are ok, but
I suggest to use Scenario Outline.
I want to see the data that you use on the Feature file.

vladimircasap commented:
> Hi Vladimir, Your codes are ok, but I suggest to use Scenario Outline. I want to see the data that you use on the Feature file.

Added negative scenario,
Added Scenario Outline
